src/test/java/org/fundacionjala/coding/abel/CodeCesarTest.java

```diff
@@ -0,0 +1,104 @@
+package org.fundacionjala.coding.abel;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+
+/**
+ *
+ */
+public class CodeCesarTest {
+
+    CodeCesar instance;
+
+    @Before
+
+    public void before() {
+        instance = new CodeCesar();
+    }
+
+
+    /**
+     *
+     */
+    @Test
+    public void test1() {
+        final int NUM2 = 31;
+        final String CAD2 = "ETWWT T";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "ETWWT T" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)